torch/distributions/logistic_normal.py

```diff
@@ -0,0 +1,46 @@
+import torch
+from torch.distributions import constraints
+from torch.distributions.normal import Normal
+from torch.distributions.transformed_distribution import TransformedDistribution
+from torch.distributions.transforms import ComposeTransform, ExpTransform, StickBreakingTransform
+from torch.distributions.utils import _sum_rightmost
+
+
+class LogisticNormal(TransformedDistribution):
+    r"""
+    Creates a logistic-normal distribution parameterized by
+    `mean` and `std` where::
```

Comment:
Could you replace `mean` and `std` with `loc` and `scale`? Feel free to fix the stale docstrings from which you copied these too :wink: